hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -617,30 +802,36 @@ private void restart(
             onMigrate(state, deserializedVersion);
         }
 
+        // TODO Update the configuration with whatever is the new latest version in state. In reality, we shouldn't
+        //      be messing with configuration during migration, but it could happen (by the file service), so we should
+        //      be defensive about it so the software is always correct, even in that very unlikely scenario.
+        //        this.configProvider.update(null);
+
         // Now that we have the state created, we are ready to create the all the dagger dependencies
-        initializeDagger(state, InitTrigger.RESTART);
+        initializeDagger(state, RESTART);
 
         // We may still want to change the address book without an upgrade. But note
         // that without a dynamic address book, this MUST be a no-op during reconnect.
-        final var stakingInfo = stateChildren.stakingInfo();
-        final var networkCtx = stateChildren.networkCtx();
-        daggerApp.stakeStartupHelper().doRestartHousekeeping(stateChildren.addressBook(), stakingInfo);
-        if (upgrade) {
-            dualState.setFreezeTime(null);
-            networkCtx.discardPreparedUpgradeMeta();
-            if (version.hasMigrationRecordsFrom(deserializedVersion)) {
-                networkCtx.markMigrationRecordsNotYetStreamed();
-            }
-        }
+        //        final var stakingInfo = stateChildren.stakingInfo();
+        //        final var networkCtx = stateChildren.networkCtx();
+        //        daggerApp.stakeStartupHelper().doRestartHousekeeping(stateChildren.addressBook(), stakingInfo);
+        //        if (upgrade) {
+        //            dualState.setFreezeTime(null);
+        //            networkCtx.discardPreparedUpgradeMeta();
+        ////            if (version.hasMigrationRecordsFrom(deserializedVersion)) {
+        ////                networkCtx.markMigrationRecordsNotYetStreamed();
+        ////            }
+        //        }
```

Comment:
Multiple comments on this file will go into the methods implementations ?

Reply:
I'm not entirely sure. I'll work on getting these lines replaced and done right, but at the moment we don't need them to at least start doing e2e testing for basic things